ZenPacks/zenoss/Dashboard/browser/resources/js/controller/DashboardController.js

```diff
@@ -72,6 +72,9 @@
             // save handler for the dialog
             win.query('button[ref="submitButton"]')[0].on('click', function() {
                 var updatedConfig = win.getFormValues();
+                if(win.down('portlet').xtype === "watchlistportlet") {
+                    updatedConfig = win.down('grid').getStore().data.items;
+                    }
```

Comment:
Javascript doesn't care, but humans _do_ care about formatting.  Need a space between the `if` and the `(` and the closing brace (`}`) should have the same indent as the `if` does.